docs/manual/docs/cmd/teams/message/message-reply-list.md

```diff
@@ -0,0 +1,36 @@
+# teams message reply list
+
+Retrieves replies to a message from a channel in a Microsoft Teams team
+
+## Usage
+
+```sh
+teams message reply list [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-i, --teamId <teamId>`|The ID of the team where the channel is located
+`-c, --channelId <channelId>`|The ID of the channel that contains the message
+`-m, --messageId <messageId>`|The ID of the message to retrieve replies for
+`-o, --output [output]`|Output type. `json|text`. Default `text`
+`--verbose`|Runs command with verbose logging
+`--debug`|Runs command with debug logging
+
+## Remarks
+
+!!! attention
+    This command is based on an API that is currently in preview and is subject to change once the API reached general availability.
+
+You can only retrieve replies to a message from a Microsoft Teams team if you are a member of that team.
+
+## Examples
+
+Retrieve the replies to a specified message from a channel of the Microsoft Teams team
```

Comment:
Changed to `Retrieve the replies from a specified message from a channel of the Microsoft Teams team` to match the text in the command help.